clio-server/src/main/scala/org/broadinstitute/clio/server/service/ServerService.scala

```diff
@@ -64,9 +64,9 @@ class ServerService private (
 
       persistenceDAO
         .getAllSince(latestUpsert)
-        .runWith(Sink.foldAsync(0) { (count, json) =>
-          logger.debug(s"Recovering document with ID ${getUpsertId(json)}")
-          searchDAO.updateMetadata(json).map(_ => count + 1)
+        .conflateWithSeed(Vector(_))(_ :+ _)
```

Comment:
Will conflate have cause variable size batches to be sent to update?  Do we want to set up something that will send fixed size batches ? Or at least a max batch size?

Reply:
Right now it'll be variable (it'll pull whatever `Vector` has managed to build up since the last pull). I'll look into setting a max size, also playing around with merging JSON in-memory if multiple records build up for the same ID before the ES pull.

Reply:
I think you can use sliding or batch to limit the total size. I think sliding would keep the batch size constant (aside from potentially the last batch after the stream is done).